src/lib/mergeThemes.ts

```diff
@@ -184,6 +184,7 @@ const mergeThemes = (...themes: IThemeInput[]): IThemePrepared => {
     fontFaces: [],
     staticStyles: [],
     icons: {},
+    provided: true,
```

Comment:
seems that we can avoid this property to be introduced - it would be enough just to check that `theme` object is not empty for theme context consumer

Reply:
removed `provided` flag